net/stubby/files/stubby.init

```diff
@@ -265,9 +232,9 @@ service_triggers()
     trigger="$(uci_get stubby global trigger)"
     delay="$(uci_get stubby global triggerdelay "2")"
 
-    if [ "$trigger" != "none" ] && [ "$trigger" != "timed" ]; then
```

Comment:
I wouldn't consider replacing formal if-then with piped-&& as a clean up. One may find big company IT to have quality rules for if-then, because it is more consistent with C, Java, and such and therefore more clearly self documenting.